docs/management/admin/trusted-apps.asciidoc

```diff
@@ -16,50 +18,56 @@ To add a trusted application:
 
 * `Name your trusted application`: Enter a name for the trusted application.
 
+* `Description`(Optional): Enter a description for the trusted application.
+
 * `Select operating system`: Select the appropriate operating system from the drop-down.
 
-* `Field`: Select the appropriate field you want to use -- `Hash`, `Path`, or (if you are adding a Windows trusted application) `Signature`.
+* `Field`: Select the appropriate field you want to use -- `Hash`, `Path`, or (if you're adding a trusted application on Windows) `Signature`.
 +
 NOTE: You can only add a single field type value per trusted application. For example, if you try to add two `Path` values, you'll get an error message. Hash values must also be valid to add the trusted application. In addition, to minimize the gaps in visibility to the Security solution, be as specific as possible in your entries. For example, combine `Signature` information with known `Paths`.
 +
 
-* `Operator`: Defaults to `is` ("equal to"). This cannot be changed.
+* `Operator`: Select an operator to define the condition:
+   - `is`: Must be exactly equal to `Value`. This operation is required for the `Hash` and `Signature` field types.   
+   - `matches`: Can include wildcards in `Value`, such as `C:\path\*\app.exe`. This option is only available for the `Path` field type.
 
 * `Value`: Enter the hash value or file path. To add an additional value, click *AND*.
 
-* `Description`(Optional): Enter a description for the trusted application.
+. Select an option in the *Assignment* section to assign the trusted application to a specific integration policy:
++
+* `Global`: Assign the trusted application to all integration policies for {endpoint-sec}.
+* `Per Policy` (Platinum+ subscription only): Assign the trusted application to one or more specific {endpoint-sec} integration policies. Select each policy in which you want the application to be trusted.
++
+NOTE: You can also select the `Per Policy` option without assigning a policy to the trusted application at this time. For example, you might want to create and review your trusted application configurations before putting them into action with a policy.
```

Comment:
```suggestion
NOTE: You can also select the `Per Policy` option without assigning a policy to the trusted application at this time. For example, you could select the `Per Policy` option if you wanted to create and review your trusted application configurations before putting them into action with a policy.
```

Reply:
How about this small modification -- it's not just selecting `Per Policy`, but it's doing that _and_ not selecting any policies. The first sentence already says those two things, so we could just reference that.
```suggestion
NOTE: You can also select the `Per Policy` option without assigning a policy to the trusted application at this time. For example, you could do this if you wanted to create and review your trusted application configurations before putting them into action with a policy.
```

Reply:
Works for me!